tests/samples.ts

```diff
@@ -215,3 +139,118 @@ export const albMultiValHeadersRequest = (): ApplicationLoadBalancerRequestEvent
       },
    });
 };
+
+export const makeAPIGatewayRequestContextIdentity = (): APIGatewayEventIdentity => {
+   return {
+      accessKey: null,
+      accountId: null,
+      apiKey: null,
+      apiKeyId: null,
+      caller: null,
+      clientCert: null,
+      cognitoAuthenticationProvider: null,
+      cognitoAuthenticationType: null,
+      cognitoIdentityId: null,
+      cognitoIdentityPoolId: null,
+      principalOrgId: null,
+      sourceIp: '12.12.12.12',
+      user: null,
+      userAgent: 'curl/7.54.0',
+      userArn: null,
+   };
+};
+
+export const makeAPIGatewayRequestContext = (params?: MakeAPIGatewayRequestEventContextParams): APIGatewayEventRequestContext => {
+   return {
+      accountId: '123456789012',
+      apiId: 'someapi',
+      authorizer: null,
+      httpMethod: params?.httpMethod ?? 'GET',
```

Comment:
See prior note about the null coalescing operator.

```suggestion
      httpMethod: params?.httpMethod || 'GET',
```

Reply:
Corrected with the withDefault function